rwslib/extras/audit_event/main.py

```diff
@@ -11,6 +11,12 @@
 from rwslib.extras.audit_event.parser import parse
 import logging
 
+# Python 3
```

Comment:
I think we can stick with int everywhere here. In Python2.7:

Python 2.7.11 (default, Jan 22 2016, 08:29:18) 
[GCC 4.2.1 Compatible Apple LLVM 7.0.2 (clang-700.1.81)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> type(int(999999999999999999999999999999999999999999999999999999999999))
<type 'long'>
>>> 

So python 2.7 auto converts an int() to a Long if required. This is also mentioned in 2.7 docs but I didn't know this before looking into it.